docs/src/index.md

```diff
@@ -8,6 +8,77 @@ For more illustrations of its usage in conjunction with other packages, the
+    - [FreqTables.jl](https://github.com/nalimilan/FreqTables.jl): Create frequency tables / cross-tabulations. Tightly integrated with DataFrames.
+    - [HypothesisTests.jl](https://juliastats.org/HypothesisTests.jl/stable/): A range of hypothesis testing tools.
+    - [GLM.jl](https://juliastats.org/GLM.jl/stable/manual/): Tools for estimating linear and generalized linear models. Tightly integrated with DataFrames.
+    - [StatsModels.jl](https://juliastats.org/StatsModels.jl/stable/): For converting heterogeneous DataFrames into homogenous matrices for use with linear algebra libraries or machine learning applications that don't directly support DataFrames. Will do things like convert categorical variables into indicators/one-hot-encodings, create interaction terms, etc.
+    - [MultivariateStats.jl](https://multivariatestatsjl.readthedocs.io/en/stable/index.html): linear regression, ridge regression, PCA, component analyses tools. Not well integrated with DataFrames, but easily used using `StatsModels`.
+- **Machine Learning**
+    - [MLJ.jl](https://github.com/alan-turing-institute/MLJ.jl): if you're more of an applied user, there is a single package the pulls from all these different libraries and provides a single, `scikit-learn` inspired API: `MLJ.jl`. MLJ provides a common interface for a wide range of machine learning algorithms. You can find a list of all supported models (both supervised and unsupervised), as well as an evaluation of the maturity of each library, [here](https://github.com/alan-turing-institute/MLJ.jl#available-models), and [some great tutorials here](https://alan-turing-institute.github.io/MLJTutorials/).  For *most* applied users, MLJ is probably a good place to start, especially if you like / are familiar with `scikit-learn`.
+    - [ScikitLearn.jl](https://scikitlearnjl.readthedocs.io/en/latest/): A Julia wrapper around the full Python scikit-learn machine learning library. Not well integrated with DataFrames, but easily used using statsmodels.jl.
+    - Deep learning: [KNet.jl](https://denizyuret.github.io/Knet.jl/stable/tutorial/#Introduction-to-Knet-1) and [Flux.jl](https://github.com/FluxML/Flux.jl). 
+- **Plotting**
+    - [`Plots.jl`](http://docs.juliaplots.org/latest/): Powerful, modern plotting library with a syntax akin to that of `matplotlib` (in Python) or `plot` (in R). [`StatsPlots`](http://docs.juliaplots.org/latest/tutorial/#Using-Plot-Recipes-1) provides `Plots.jl` with recipes for many standard statistical plots.
+    - [`Gadfly.jl`](http://gadflyjl.org/stable/): High-level plotting library with a "grammar of graphics" syntax akin to that of `ggplot` (in R).
+    - [`VegaLite.jl`](https://www.queryverse.org/VegaLite.jl/stable/): High-level plotting library that uses a different "grammar of graphics" syntax and has an emphasis on interactive graphics.
+- **Data Wrangling**:
```

Comment:
Note that DataFramesMeta and Query are already mentioned in the "Data Management Frameworks" page. Put a link to there?